build/template-android-appcenter-tests.yaml

```diff
@@ -3,17 +3,19 @@ parameters:
 
 steps:
 
-- task: DownloadBuildArtifacts@0
-  displayName: 'Download Drop'
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Xamarin.Android'
   inputs:
-    artifactName: drop
-    itemPattern: '**/*'
+    artifact: Xamarin.Android
+    patterns: '**/*'
+    path: $(System.ArtifactsDirectory)/Xamarin.Android
```

Comment:
Any reason why Android is dropped into a separate folder? IOS stuff is dropped into root artifact directory.

Reply:
Bug, as I can't start an iOS build to validate. Fixed now, thanks!